### ISSUE TITLE
Need feedback on test-queue.

### ISSUE THREAD

**Issue by chadsansing** [mozilla/learning-networks#242]
Please visit http://chadsansing.github.io/curriculum-testing/curriculum-queue/ and use this thread for feedback. The queue will need CSS help, but are these the right modules to develop? Are they in the right sequence?

Please submit feedback by EOD, Friday, September 18th, 2015.

Many thanks!

**Comment by jgmac1106** [comment]
My only thinking is we may want to wait on the que until @An-me's focus
groups for web literacy are over. Even with a  cursory look there are
patterns across all stakeholders in terms of curriculum they want to see.

Also given that the web literacy map development is no longer being done in
the open we have no idea what competencies will be in the final product.
Therefore its hard to say how the corresponding curriculum should be
developed.

**Comment by chadsansing** [comment]
Points taken, but I anticipate at least some alignment. Wondering about what is absolutely blocked and what can go forward. Eager for the reboot.

**Comment by chadsansing** [comment]
cc @amirad, @omnignorant, @anmechung, @chrislarry33, @ldecoursy, @hannahkane, Leah.

I'd like to gather feedback ASAP and move ahead with a reboot of the queue before next week's team call.

**Comment by jgmac1106** [comment]
Is Web Literacy Basics II Teaching kit still the Privacy Activities? Did
you remove the JS Teaching Kit or are those activities folded into the
Intermediate Web Development kit?

The order makes sense.

Sorry for the confusion  I can't remember where we left off with the
curriculum calls.

Not sure how far out in 2016 you want to go but I think adding a
Credibility kit would make sense.

I also think we should design a Teaching Kit specifically for club
launching down the road..

**Comment by hannahkane** [comment]
Same question as @jgmac1106 about where Protect Your Data module fits in (it's currently the second kit on https://teach.mozilla.org/activities, but isn't labeled "Web Lit Basics II." We could rename it.)

Also wondering if the current Lo fi/No fi kit (third on that same list) will be replaced by the "Teach the Web Offline" kit in the queue.

**Comment by chadsansing** [comment]
I think "Protect Your Data" stays "Protect Your Data." "Web Literacy Basics II" will be scoped & likely provide more expanded, but broadly basic HTML, CSS, & JS practice (might also be the place for multi=page project development and GitHub tutorials).

I think "Teach the Web Offline" will be more exclusively no-fo than the other kit. 

Thoughts, @amirad?

**Comment by jgmac1106** [comment]
Thanks for clarification.

I think this makes more sense than having protect your data listed as Web
Literacy Basics II (Just checked the live page Protect your data no longer
says Web Literacy Basics II). I would hope that after you finish Basics I
and II you cover all the competencies in the map.

I think we will need to add design and accessibility activities in Web
Literacy Basics II as we don't hit those enough. I don't think we got to
that enough in Web Lit Basic 1.

Determining the line between basic and intermediate will be a hard but fun
delineation to make

**Comment by leahatplay** [comment]
Are established Hive-y practices like Hive Meet-ups, Deep Dives (CHI), Project Learning Labs(NYC) etc. included in the Mobilizing Community, Working Locally etc. "Leveraging Open Practices" work?

It would be a big win to get basic open practices like these as TLM components prior to the scale/spread work which is pretty high-level. 

This needs updating but is a great resource:  How to host a hive meet-up: https://juliahivenyc.makes.org/thimble/LTE3MjUzNjYwMTY=/how-to-host-a-hive-meet-up 

Also: Project Learning Lab and process behind sharing around projects documented here: hivenycohort.tumblr.com

**Comment by chadsansing** [comment]
@leahatplay - all of the suggested TLM modules would draw from the Hive resources you shared with me; those would be anchors of practice, and we could connect them to work in other MLN spaces, as well.
